invenio_rdm_migrator/streams/streams.py

```diff
@@ -11,11 +11,26 @@
 from datetime import datetime
 
 
+class StreamDefinition:
+    """ETL stream definition.
+
+    It defines the classes that will form the ETL stream.
+    """
+
+    def __init__(self, name, extract_cls, transform_cls, load_cls):
```

Comment:
self doubt: wondering if this is better or we should go with config classes like we do in the mvc layers (e.g. `ServiceConfig`).